collectd-docker.py

```diff
@@ -222,6 +222,10 @@
     'docker-librato.\w+.cpu_stats.cpu_usage.percpu_usage.*',
 ]
 
+whitelist = [re.compile(l) for l in WHITELIST_STATS]
+blacklist = [re.compile(l) for l in BLACKLIST_STATS]
```

Comment:
The above constants don't really need to be constants, since they aren't referenced anywhere but here.